plugins/apache-airflow/src/components/DagTableComponent/DagTableComponent.tsx

```diff
@@ -147,5 +162,18 @@ export const DagTableComponent = () => {
     dagUrl: `${apiClient.baseUrl}dag_details?dag_id=${el.dag_id}`, // construct path to DAG using `baseUrl`
   }));
 
-  return <DenseTable dags={data || []} />;
+  return (
+    <>
+      {dagsNotFound ? (
+        <WarningPanel title={`${dagsNotFound.length} DAGs were not found`}>
+          {dagsNotFound.map(dagId => (
+            <Typography>{dagId}</Typography>
+          ))}
+        </WarningPanel>
+      ) : (
+        ''
```

Comment:
does this need to be an empty string? Otherwise you should be able to do `{dagsNotFound  && <Warning...`

Reply:
Good catch! 😅

Reply:
I believe this got reset somehow 😀